js/src/redux/reducers/index.js

```diff
@@ -1,17 +1,21 @@
-import { analysis } from "yoast-components";
 import activeMarker from "./activeMarker";
 import advancedSettings from "./advancedSettings";
+import { analysis } from "yoast-components";
 import analysisData from "./analysisData";
 import editorContext from "./editorContext";
 import editorData from "./editorData";
 import estimatedReadingTime from "./estimatedReadingTime";
-import isCornerstone from "./cornerstoneContent";
-import isMarkerPaused from "./markerPauseStatus";
 import facebookEditor from "./facebookEditor";
 import focusKeyword from "./focusKeyword";
+import isCornerstone from "./cornerstoneContent";
+import isMarkerPaused from "./markerPauseStatus";
 import marksButtonStatus from "./markerButtons";
 import preferences from "./preferences";
 import primaryTaxonomies from "./primaryTaxonomies";
+import schemaBlocks from "./schemaBlocks";
+import schemaTab from "./schemaTab";
```

Comment:
Also: Travis fails since `schemaTab` is imported twice in this file. Here and on line 23.

Reply:
ah, merge conflict :-D